src/components/ResourceBrowser/ResourceList/EventList.tsx

```diff
@@ -31,20 +30,19 @@ export const EventList = ({
     searchText,
 }: EventListType) => {
     return (
-        <div>
-            <div className="event-list-row fw-6 cn-7 fs-13 dc__border-bottom pl-20 pr-8 pt-8 pb-8 dc__uppercase h-36">
-                <div>{EVENT_LIST.headerKeys.type}</div>
-                <div>{EVENT_LIST.headerKeys.message}</div>
-                <div>{EVENT_LIST.headerKeys.namespace}</div>
-                <div>{EVENT_LIST.headerKeys.involvedObject}</div>
-                <div>{EVENT_LIST.headerKeys.source}</div>
-                <div>{EVENT_LIST.headerKeys.count}</div>
-                <div>{EVENT_LIST.headerKeys.age}</div>
-                <div>{EVENT_LIST.headerKeys.lastSeen}</div>
+        <div className="dc__overflow-scroll">
+            <div className="event-list-row dc__zi-1 dc__min-width-fit-content dc__position-sticky bcn-0 dc__top-0 fw-6 cn-7 fs-13 dc__border-bottom pl-20 pr-8 pt-8 pb-8 dc__uppercase h-36">
+                {Object.values(EVENT_LIST.headerKeys).map((title) => (
+                    <Tooltip showOnTruncate content={title}>
```

Comment:
I was wrong. The key should be on Tooltip. Apparently the immediate children of a map should be given keys